tests/reporter/kubernetes.go

```diff
@@ -66,6 +66,7 @@ const (
 const (
 	k8sCNICNCFEntityURLTemplate       = "/apis/k8s.cni.cncf.io/v1/namespaces/%s/%s/"
 	networkAttachmentDefinitionEntity = "network-attachment-definitions"
+	mountNamespace                    = "/proc/1/ns/mnt"
```

Comment:
This will not work for container-based Kubernetes, right?

Reply:
@xpivarc can you please elaborate more ? We are new to kubevirt world and need to understand what you mean by container based Kubernetes and why it will not work ? The work done in this PR is for k3s running in a container and kubevirt pods running in that container.

Reply:
@xpivarc I'm trying to see if this tests/reporter is used on local testing or this code is running at kubevirt runtime. If former, we don't need to handle that, and there is no need to setup the kubevirt inside a container for testing.

Reply:
Hey, sorry for the late reply, I missed the notification.

While running tests on your setup (k3s) you would need to use something else than `/proc/1/ns/mnt`. You can verify it by running Kubevirt with this patch on your setup and executing the tests requiring this variable.

Reply:
@xpivarc my environment is on an edge device with  limited resources, with Alpine linux. We have a container running k3s/kubevirt/longhorn etc those kubernetes services inside this container. I'm not sure if it is possible to run your kubeveirt test inside this kind of container.  There is no kubevirt repo to 'make' or 'make test' inside my container environment.

Reply:
Well, you can run the tests against external/not local cluster as well. What I mean by my comment is that if you try it, it will fail because the tests will not use the correct namespace.

Reply:
@xpivarc updated the patch, and please review.